### ISSUE TITLE
AutoAugment is not very efficient on my 5000 item dataset?

### ISSUE THREAD

**Issue by fitoule** [google/automl#481]
![augment](https://user-images.githubusercontent.com/35316806/83879226-97cf2480-a73d-11ea-8244-6ccc533245c5.jpg)

Hi I tried on my dataset training EfficientDet D2 3 classes about 5000 images training (finetuning from D2 Cocodataset). When I try autoaugment policy V2 or V3 it is always not as so good as the raw training without any augment :( How cxan I resolve that ?

Thank you

**Comment by mingxingtan** [comment]
Yeah, I am not sure how effective of those autoaugmentation: they might depend on the dataset and so on.

For COCO, I found autoaugment needs longer training, so I ended up not using them for COCO.

**Comment by Koteswara-ML** [comment]
Hey, Did you train on TPU..for training using augmentations, Can you help me with your training parameters .
